app/src/main/java/cm/aptoide/pt/home/apps/list/models/UpdateCardModel.kt

```diff
@@ -120,7 +123,8 @@ abstract class UpdateCardModel : EpoxyModelWithHolder<UpdateCardModel.CardHolder
       holder.secondaryText.setTextAppearance(holder.itemView.context,
           R.style.Aptoide_TextView_Medium_XS_Red700)
     } else {
-      holder.secondaryIcon.setImageResource(R.drawable.ic_refresh_orange)
+      holder.secondaryIcon.setImageResource(
+          themeManager!!.getAttributeForTheme(R.attr.version_refresh_icon).resourceId)
```

Comment:
Not a big deal but this is not very safe. If ThemeManager is really required, maybe make it 'lateinit var' instead?

Or, if it's not crucial, maybe use elvis operator for it, i.e.:
`themeManager?.getAttributeForTheme(R.attr.version_refresh_icon)?.resourceId ?: R.drawable.ic_refresh_action_white`

Reply:
done